src/App.js

```diff
@@ -1,112 +1,25 @@
-// import * as React from 'react';
-
-// const todoList = [
-//   {
-//     title: 'The React Task 1.1',
-//     id: 1,
-//   },
-//   {
-//     title: 'Submission Task 1.1',
-//     id: 2,
-//   },
-// ];
-
-// function TodoList (){
-//   return (
-//     <ul>
-//       {todoList.map(function (item) {
-//         return(
-//         <li key={item.objectID}>
-//           <span>
-//             <a href={item.url}>{item.title}</a>
-//           </span>
-//           <span>{item.author}</span>
-//           <span>{item.num_comments}</span>
-//           <span>{item.points}</span>
-//         </li>
-//       );
-//       })}
-//       </ul>
-//   )
-// }
-
-// function Search (){
-//   return(
-//     <div>
-//       <label htmlFor = "search"> Search: </label>
-//       <input id = "search" type = "text" />
-//     </div>
-//   )
-// }
-
-// function App() {
-//   return (
-//   <div>
-//     <h1>Todo List</h1>
-//     <Search />
-//     <Search />
-
-//       <hr />
-//       <TodoList />
-//       <TodoList />
-      
-//     </div >
-//   );
-// }
-
-// export default App;
 
 import * as React from 'react';
 import TodoList from './TodoList';
-import AddTodoForm from './AddTodoForm'
-// import AddTodoForm from './AddTodoForm'
+import AddTodoForm from './AddTodoForm';
+import Search from './search';
```

Comment:
The import of the Search component should be written as import Search from './Search';. Please pay attention to file and component names when importing.

Reply:
done